hdk-rust/src/api/get_entry.rs

```diff
@@ -75,10 +75,10 @@ pub fn get_entry_history(address: &Address) -> ZomeApiResult<Option<EntryHistory
 /// The data returned is configurable with the GetEntryOptions argument.
 pub fn get_entry_result(
     address: &Address,
-    options: GetEntryOptions,
+    options: &GetEntryOptions,
 ) -> ZomeApiResult<GetEntryResult> {
     Dispatch::GetEntry.with_input(GetEntryArgs {
         address: address.clone(),
```

Comment:
So I think I'm confused about this PR.  In the [checklist CALLER-CONTROL](https://rust-lang-nursery.github.io/api-guidelines/flexibility.html#c-caller-control) seems to indicate that we shouldn't be doing exactly this.  Am I mis-understanding?